client/erc721/mainnet_erc721_contract.go

```diff
@@ -48,6 +48,18 @@ func (c *MainnetERC721Client) OwnerOf(tokenID *big.Int) (common.Address, error)
 	return c.contract.OwnerOf(nil, tokenID)
 }
 
+func (c *MainnetERC721Client) Exists(tokenID *big.Int) (bool, error) {
+	return c.contract.Exists(nil, tokenID)
+}
```

Comment:
You removed this public `exists` function in the Solidity contract, didn't you?

Reply:
yea I removed the function. This should be removed too